pubspec.yaml

```diff
@@ -1,5 +1,5 @@
 name: json_schema
-version: 2.2.0
+version: 2.2.1
```

Comment:
If you wouldn't mind backing this change out, that would be great. Our internal release bots will increment on release.

Reply:
Done